api-extractor.json

```diff
@@ -400,7 +400,7 @@
         "addToApiReportFile": false
       },
       "ae-incompatible-release-tags": {
-        "logLevel": "error",
+        "logLevel": "warning",
```

Comment:
@adrianschmidt I think we need to change this to `warning`. Stencil does not seem to output all the doc blocks, which makes API extractor complain about the release tags not being compatible

Reply:
The way to "ignore" it, is to set `addToApiReportFile` to `true`. Then it adds the warning to the api report instead of outputting it on the command line, and that also makes the verify-check pass (at least if it's set to "warning").

That's what I've done with `ae-missing-release-tag` below (line 410-421).

Then we still get the warning in the api report, and can at least review it as part of the PR.